src/Middleware/ParamConverterMiddleware.php

```diff
@@ -64,31 +64,34 @@ public function handle(Request $request, Closure $next)
         // Resolve the controller callable based on the request.
         $controller = $this->resolveController($route);
 
-        if ($controller !== null) {
-            // Add laravel route attributes to the symfony request attribute bag so the
-            // symfony dependencies will work as expected.
-            $request->attributes->add($route[2]);
+        if ($controller === null) {
+            // Controller is not callable, continue with other middleware
+            return $next($request);
+        }
 
-            // Create a faux FilterControllerEvent for use in the symfony dependencies below.
-            $filterController = new FilterControllerEvent($controller, $request);
+        // Add laravel route attributes to the symfony request attribute bag so the
+        // symfony dependencies will work as expected.
+        $request->attributes->add($route[2]);
```

Comment:
I love magic numbers, especially in array lookups.
(I totally understand that this is because of the asinine way that Laravel passes this data along, but it's just so horrible that I feel the need to comment on it.)